plugins/net_plugin/net_plugin.cpp

```diff
@@ -1391,10 +1404,9 @@ namespace eosio {
 
    void sync_manager::send_handshakes()
    {
-      for( auto &ci : my_impl->connections) {
-         if( ci->current()) {
-            ci->send_handshake();
-         }
+      auto itr = my_impl->connections.find(connection_state::syncing);
+      for (; itr != my_impl->connections.end(); ++itr) {
```

Comment:
Where is checking on `syncing`?

Reply:
@afalaleev default index is sorted by state, so all of them are syncing.